### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by TheLiao233** [UTSJiyaoLi/Adversarial-Image-Captioning-Attack#1]
Hello, I would like to ask how should I get the `attens_10samples.json` file of `pick_atten.py `and the `attens_100samples_255.json` of `perturb_att.py`.
If you can, Could you please provide the `BEST_checkpoint_coco_5_cap_per_img_5_min_word_freq.pth.tar` and `WORDMAP_coco_5_cap_per_img_5_min_word_freq.json` of `get_att.py` 
Sincerely hope for your reply, thank you!

**Comment by UTSJiyaoLi** [comment]
The files `attens_10_samples.json` and `attens_100samples_255.json` can be generated from `get_att.py`. The exact file names depend on the settings you configure in the `get_att.py`.

To help you run the code, I have also uploaded some sample `attens_... .json` files. Additionally, I have updated the `README.md` to include a link for downloading the model file `BEST_checkpoint_coco_5_cap_per_img_5_min_word_freq.pth.tar`.

The `WORDMAP_coco_5_cap_per_img_5_min_word_freq.json` file has also been uploaded for your convenience.

You can use the following steps to generate or use these files:

1. Configure and run the `get_att.py` script based on your desired settings.
2. Refer to the uploaded `.json` files for sample attention results.
3. Download the pre-trained model from the link provided in `README.md`.

Please ensure that you have the following dependencies set up before running the codes:
- `BEST_checkpoint_coco_5_cap_per_img_5_min_word_freq.pth.tar` (pre-trained model)
- `WORDMAP_coco_5_cap_per_img_5_min_word_freq.json` (word map file)

**Comment by TheLiao233** [comment]
> The files `attens_10_samples.json` and `attens_100samples_255.json` can be generated from `get_att.py`. The exact file names depend on the settings you configure in the `get_att.py`.
> 
> To help you run the code, I have also uploaded some sample `attens_... .json` files. Additionally, I have updated the `README.md` to include a link for downloading the model file `BEST_checkpoint_coco_5_cap_per_img_5_min_word_freq.pth.tar`.
> 
> The `WORDMAP_coco_5_cap_per_img_5_min_word_freq.json` file has also been uploaded for your convenience.
> 
> You can use the following steps to generate or use these files:
> 
>     1. Configure and run the `get_att.py` script based on your desired settings.
> 
>     2. Refer to the uploaded `.json` files for sample attention results.
> 
>     3. Download the pre-trained model from the link provided in `README.md`.
> 
> 
> Please ensure that you have the following dependencies set up before running the codes:
> 
>     * `BEST_checkpoint_coco_5_cap_per_img_5_min_word_freq.pth.tar` (pre-trained model)
> 
>     * `WORDMAP_coco_5_cap_per_img_5_min_word_freq.json` (word map file)

Thank you very much for your timely reply.  There are still some problems: 

1.Does each `samples.json` used in the code differ only in `num_input`, and what are the implications of using different files?  The `flicker8k` dataset is used by default in `get_att.py` and the generated file has 384 in its name, but several other json files don't have enough information in their names to make it impossible to determine how to generate with the appropriate parameters. In particular, whether files with 255 in their names need to adjust the code of `get_att.py` ？

Specific documents are as follows：

- `attens_10samples.json` is used in the `pick_atten.py`
- `attens_100samples.json` is used in the `main.py`
- `attens_100samples_255.json` is used in the `perturb_att.py`
- `attens_flicker8k_1000samples_255.json` is used in the `separate.py`

2.After I generated the above files myself, the running result of `attack_blip_test.py `is as follows. The loss and generated text are basically unchanged, is there any problem?

![图片](https://github.com/user-attachments/assets/22a49c31-f753-4d34-bff3-78a474551aa6)

**Comment by UTSJiyaoLi** [comment]
The BLIP model takes an input image size of 384 * 384, and SAT takes 255 * 255. Check the parameters you used for attacking BLIP, such as `pixels`.

**Comment by Summer2307** [comment]
> 2.After I generated the above files myself, the running result of `attack_blip_test.py `is as follows. The loss and generated text are basically unchanged, is there any problem?
> 
> ![图片](https://github.com/user-attachments/assets/22a49c31-f753-4d34-bff3-78a474551aa6)

Hi :) 
I encountered the similar problem of the loss and generated texts nearly unchanged, may I ask how did you solve it? Thanks a lot!

**Comment by NguyenNgocThach2001** [comment]
> > 2.After I generated the above files myself, the running result of `attack_blip_test.py `is as follows. The loss and generated text are basically unchanged, is there any problem?
> > ![图片](https://github.com/user-attachments/assets/22a49c31-f753-4d34-bff3-78a474551aa6)
> 
> Hi :) I encountered the similar problem of the loss and generated texts nearly unchanged, may I ask how did you solve it? Thanks a lot!


~~I think it's because BLIP is robust against this attack method. You could try increasing the number of attack pixels and printing the image.~~

~~With a 5000 pixel attack, the BLEU score still does not change, but you can see that the image is crowded with noise.~~

<img width="445" height="131" alt="Image" src="https://github.com/user-attachments/assets/2f5985b3-ed27-489e-b14a-3fb5c82b07b6" />

![Image](https://github.com/user-attachments/assets/e999a7e2-643d-4e8b-a0af-c2c806ffcc2d)

Edit1: I might be wrong, I might use the wrong ref image, let me check again
Edit2: Checking if the reference caption loaded correctly. Since both "old" and "new" results differ significantly from the ref_caption, explaining the low BLEU score
